### PR TITLE
[CN-671] Add page with JVM memory parameters

### PR DESCRIPTION
**Pull request by dzeromski-hazelcast**

### CODE REVIEW

File: docs/modules/ROOT/pages/jvm-parameters.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+= JVM Parameters
+:description: pass:q[Hazelcast allows selected JVM arguments to be passed to the JVM in the `hazelcast` container by using th `jvm` field of the CRD spec.]
+
+{description}
+
+== Configuring a JVM Parameter
+
+You can use any of the following configuration options in the `Hazelcast` resource. For more information, see the xref:api-ref.adoc#hazelcast[API Reference].
```
Comment (mtyazici) [inline docs/modules/ROOT/pages/jvm-parameters.adoc:8]: I think this is more fitting.
```suggestion
You can use any of the following configuration options in the `Hazelcast` resource. For more information, see the xref:api-ref.adoc#jvmconfiguration[API Reference].
```

File: docs/modules/ROOT/pages/jvm-parameters.adoc [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+[cols="25%m,75%a"]
+|===
+|Field|Description
+
+|name
+|Name of the JVM Memory parameter.
+
+|initialRAMPercentage
```
[the remark this answers is not in the document]
Comment (mtyazici) [reply docs/modules/ROOT/pages/jvm-parameters.adoc:14]: I don't know if it will create confusion on the user side

**Review by mtyazici — APPROVED** [review]

File: docs/modules/ROOT/nav.adoc [hunk 3]
```diff
@@ -18,6 +18,7 @@
 .Reference
 // Configuration options/spec files/any other reference docs
 * xref:phone-homes.adoc[Phone Homes]
+* xref:jvm-parameters.adoc[JVM Parameters]
```
Comment (hasancelik) [inline docs/modules/ROOT/nav.adoc:21]: Why this page is listed under `Reference`? I think it should be listed with other features at `Get Started`, right?

Comment (angelasimms) [reply docs/modules/ROOT/nav.adoc:21]: @hasancelik. If you decide to move the page, could you change the title to `Configuring JVM Parameters`?

**Review by angelasimms — CHANGES_REQUESTED** [review]
Just minor rewording.

File: docs/modules/ROOT/nav.adoc (hunk 3 above)
Comment (hasancelik) [reply docs/modules/ROOT/nav.adoc:21]: @dzeromski-hazelcast I think we should move it. Could you change the title also? :)

**Review by hasancelik — APPROVED** [review]

**Pull request merged**